addons/mail/static/src/components/link_preview_aside/link_preview_aside.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module **/
+
+import { registerMessagingComponent } from '@mail/utils/messaging_component';
+
+const { Component } = owl;
+
+class LinkPreviewAside extends Component {
```

Comment:
All new components should have View suffix to match their corresponding model name